src/js/dbSNP-import/dbSNP-import-progress.js

```diff
@@ -90,39 +93,38 @@ EvadbSNPImportProgress.prototype = {
                 '</tr>' +
                 '</thead><tbody>';
 
-        //data = _.sortBy(data, 'importedIds', 'commonName');
         _.each (_.keys(data), function(key) {
             var genbankAssemblyAccession = '-';
             var taxonomy_link;
 
-            if(!_.isNull(this[key].genbankAssemblyAccession)){
-                genbankAssemblyAccession = '<a target="_blank" href="https://www.ebi.ac.uk/ena/data/view/'+this[key].genbankAssemblyAccession+'">'+this[key].genbankAssemblyAccession+'</a>';
+            if(!this[key].genbankAssemblyAccession.isEmpty()) {
```

Comment:
I can see some nulls in the DB, will they be transformed into empty strings by spring or javascript?

Reply:
The `genbank_assembly_accession` column has been modified in the database to be `not null` so we will have `empty strings` in those cases. This change was made in the development database but is not yet reflected in the production one.